pages/md-style-guide.md

```diff
@@ -1,191 +1,165 @@
 
----
+<blockquote> 
+ Don't use too many notices. When you use multiple notices on a page, they begin to lose their visual distinctiveness. See if you can convey the information in a different way. This is especially true if you have two (or more) notices in a row.
+ </blockquote>
 
-## Infobox
+### Tabs group
 
-Infobox is useful for presenting an information, inside a box. The text written between these tags will be shown.
+Use the tabs groups to make comparisons. 
 
-```markdown
-<Infobox label="label"> message to show. </Infobox>
-```
+`[ tabs-start "label"]
+[ tab "name"]
+message to show
+[ tabs-end ]`
 
-### Example
+Example:
 
-<Infobox label="label"> message to show. </Infobox>
+[tabs-start "Language-specific info"]
 
----
+[tab "Python"]
+In Python, "objects" are analogous to the `dict` type.  An
+important difference, however, is that while Python dictionaries
+may use anything hashable as a key, in JSON all the keys
+must be strings.
 
-## Warning
+Try not to be confused by the two uses of the word "object" here:
+Python uses the word `object` to mean the generic base class for
+everything, whereas in JSON it is used only to mean a mapping from
+string keys to values.
 
-Warning is a special type of info box in which text showing any warning can be shown. The text written between these tags will be shown.
+[tab "Ruby"]
+In Ruby, "objects" are analogous to the `Hash` type. An important
+difference, however, is that all keys in JSON must be strings, and therefore
+any non-string keys are converted over to their string representation.
 
-```markdown
-<Warning label="label"> message to show. </Warning>
-```
+Try not to be confused by the two uses of the word "object" here:
+Ruby uses the word `Object` to mean the generic base class for
+everything, whereas in JSON it is used only to mean a mapping from
+string keys to values.
 
-### Example
+[tab "Objective-C"]
+In Objective-C, "objects" are analogous to the `NSDictionary` type.
 
-<Warning label="label"> message to show. </Warning>
+[tab "Swift"]
+In Swift, "objects" are analogous to the `Dictionary` type.
+[tabs-end]
 
----
+### Star tag
 
-## Tip
+Highlight changes introduced by drafts with the start tag.
 
-Tip is a special type of info box in which text showing a tip can be shown. The text written between these tags will be shown.
+`<Star label="New in draft"/>`
 
-```markdown
-<Tip label="label"> message to show. </Tip>
-```
+Example:
 
-### Example
+<Star label= "New in draft 2019-09"/>
 
-<Tip label="label"> message to show. </Tip>
+The deprecated keyword is a boolean that indicates that the instance value the keyword applies to should not be used and may be removed in the future.
 
----
+### Infobox tag
 
-## Danger
+Use the infobox tag for [Notes](https://developers.google.com/style/notices). Notes provide the reader with relevant information that is not necessary, for example, an aside or a tip. 
```

Comment:
Would the word 'essential' be suitable in this context -

'Notes provide the reader with relevant information that is not **essential**, for example, an aside or a tip.'